### DISCUSSION TITLE
Quick settings

### DISCUSSION THREAD

**Discussion by katanaur** [felixwiemuth/SimpleReminder#45] [Questions] [answered]
I see there is some Quick Settings mention in help and change log, but i can't see it anywhere in the app (0.9.12, android 10).

**Comment by felixwiemuth** [comment, marked as answer]
Hi! The Quick Settings are part of your devices general user interface, you find them in the drawer where also notifications are.

More details here: https://support.google.com/android/answer/9083864?hl=en

I hope this helps and you can find the SimpleReminder icon :)

**Reply by katanaur** [reply]
Oh, that quick settings. I thought it is something in the app itself :)
Thanks!
